collectd_opentsdb_plugin.py

```diff
@@ -0,0 +1,62 @@
+import os
+import potsdb
+import collectd
+
+
+class OpenTSDBExportPlugin:
+
+    def __init__(self):
+        self.metrics = None
+
+    def configure_callback(self, conf):
+        for node in conf.children:
+            key = node.key.lower()
+            value = node.values[0]
+            if key == 'host':
+                self._opentsdb_host = value
+            elif key == 'port':
+                self._opentsdb_port = value
+        if not self._opentsdb_host or not self._opentsdb_port:
+            raise Exception("OpenTSDB export plugin is not configured")
+        collectd.info("Configured OpenTSDB export plugin (%s:%s)" %
+                      (self._opentsdb_host, self._opentsdb_port))
+
+    def init_callback(self):
+        self.metrics = potsdb.Client(host=self._opentsdb_host,
+                                     port=self._opentsdb_port,
+                                     mps=100, check_host=True)
```

Comment:
`check_host=True` means client failure if opentsdb is unreacheable at that moment, but it pass and later [silently retry connection](https://github.com/orionvm/potsdb/blob/master/potsdb/client.py#L30) while events are [queued up to 100k messages](https://github.com/orionvm/potsdb/blob/master/potsdb/client.py#L82).

In our case, when it fails to start due to `check_host=True`, Marathon will take care of relaunching unless collectd doesn't have such logic already.

The lack of backoff in retries at [postdb/client.py#L29](https://github.com/orionvm/potsdb/blob/master/potsdb/client.py#L29) worries me more, in case of failure it will enter a no-delay loop.